app/assets/javascripts/components/admins/pages/interviews/editable_input.jsx

```diff
@@ -16,9 +16,17 @@ class EditableInput extends React.Component {
         }
```

Comment:
Won't let me comment on areas you didn't touch :/

But instead of defining our own toggleEdit here, we can modify the statement above in the else case to return

```
                <div onClick={this.props.toggleEdit}>
                    { this.props.data }
                </div>
```

and then we can remove the_toggleEdit here and remove it from the onClick of the fieldset